tests/snmpsim/generex-ups-card.snmprec

```diff
@@ -0,0 +1 @@
+1.3.6.1.2.1.1.1.0|4|CS121
```

Comment:
sysDescr on mine is STRING: "CS121 v 5.14.50"
sysObjectId on mine is OID: RFC1213-MIB::mib-2.33
@FTBZ can you confirm your sysDescr and sysObjectId. I have a feeling they will vary by firmware version but hopefully nothing more.